README.md

```diff
@@ -27,20 +37,48 @@ be integrated with other applications and languages utilizing its web service.
 * QuickBooks is supported by XBRL-GL.
 
 
-## Installation
+# Installation
 
-Please check out our [Arelle install guide](INSTALLATION.md).
+The implementation is in Python 3.10, and is intended for Windows (any recent),
+Mac OS-X 10.9+, Unix or Linux. The standard installation includes both desktop GUI,
+a localhost RESTful web server and CLI.
```

Comment:
* We don't officially support Python 3.10 yet. Hopefully within the next week that'll change.
* Preexisting from the installation guide - supported OS versions are out of date

```suggestion
The implementation is in Python 3.9 and is intended for Windows, macOS, or Linux (tested against Ubuntu and RHEL). The standard desktop installation includes a GUI, a RESTful web server, and CLI. We try to support all operating system versions that still receive security updates from their development teams.
```